Editor/Builders/AssetsItemBuilder.cs

```diff
@@ -37,6 +38,12 @@ public AdvancedDropdownItem Build()
                 {
                     CreateItemForPath(root, assetPath);
                 }
+                else if (assetType == typeof(GameObject))
+                {
+                    GameObject gameObject = AssetDatabase.LoadAssetAtPath<GameObject>(assetPath);
```

Comment:
Hmm. I have a feeling that this might become very slow for larger projects

Reply:
Oof yeah that is pretty bad! I'll have a look at your PR!

Reply:
Yeah, sorry about that! It became so slow in our project that we had to comment it out. We were looking in a way to stream the assets instead of loading them. I believe the new Search package in Unity does that, but I'm not sure how to use their API yet.